src/api/java/com/minecolonies/api/quests/IQuestData.java

```diff
@@ -0,0 +1,67 @@
+package com.minecolonies.api.quests;
+
+import com.minecolonies.api.colony.IColony;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.player.Player;
+
+import java.util.List;
+
+/**
+ * Quest Data Instance.
+ */
+public interface IQuestData
```

Comment:
Wanted to avoid IQuest, because else it could seem that a IQuest is assigned to a colony. This is just a global data holder without contextual quest data yet.